package.json

```diff
@@ -10,14 +10,16 @@
     "test": "turbo run test",
     "clean": "turbo clean",
     "format": "prettier --write .",
-    "format-check": "prettier --check ."
+    "format-check": "prettier --check .",
+    "host": "turbo run build --filter=webapp && turbo run host --filter=webapp"
```

Comment:
This line is more relevant to the specific app. I would not recommend putting this at the top level of the monorepo.

Reply:
should we call pnpm host from apps/webapp? not from root

Reply:
Yes, because the way the app is hosted is not a concern to the monorepo, but to that app specifically.